tensornetwork/backends/pytorch/pytorch_backend_test.py

```diff
@@ -110,3 +111,105 @@ def test_outer_product():
   expected = np.array([[[[[2.0, 2.0], [2.0, 2.0]]]], [[[[2.0, 2.0], [2.0,
```

Comment:
Can you do `np.array([2.0] * 8).reshape((1, 1, 2, 2, 2))`  instead? Or whatever shape it is. I can't tell right now which is why you should change it. It also helps avoid the ugly `[[[[`